rbac/management/workspace/view.py

```diff
@@ -50,6 +52,7 @@ class WorkspaceViewSet(
     queryset = Workspace.objects.annotate()
     lookup_field = "uuid"
     serializer_class = WorkspaceSerializer
+    renderer_classes = api_settings.DEFAULT_RENDERER_CLASSES + [ProblemJSONRenderer]
```

Comment:
This ensures we pull in the default JSON renderer and just add the custom problem renderer. We'll need to do this for each v2 view, _or_ design a better way to inherit from a v2 viewset, have middleware plug this in based on path, etc. I figured this was probably the easiest for now, and we can revisit if folks think we should refactor when we introduce more views.

Reply:
Look to see if we can have a v2 viewset to inherit from.